
(* This file was auto-generated based on "ConcreteSyntax_Parser.messages". *)

(* Please note that the function [message] can raise [Not_found]. *)

let message =
  fun s ->
    match s with
    | 0 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 29\n"
    | 1 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 28\n"
    | 5 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 27\n"
    | 6 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 26\n"
    | 9 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 25\n"
    | 4 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 24\n"
    | 54 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 23\n"
    | 22 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 22\n"
    | 25 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 21\n"
    | 26 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 20\n"
    | 27 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 19\n"
    | 34 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 18\n"
    | 31 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 17\n"
    | 35 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 16\n"
    | 28 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 15\n"
    | 40 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 14\n"
    | 41 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 13\n"
    | 60 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 12\n"
    | 58 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 11\n"
    | 62 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 10\n"
    | 12 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 9\n"
    | 64 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 8\n"
    | 19 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 7\n"
    | 20 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 6\n"
    | 47 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 5\n"
    | 13 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 4\n"
    | 15 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 3\n"
    | 68 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 2\n"
    | 66 ->
        "<YOUR SYNTAX ERROR MESSAGE HERE> 1\n"
    | _ ->
        raise Not_found
